cartography/intel/gcp/bigquery.py

```diff
@@ -277,6 +346,22 @@ def cleanup_gcp_bigquery(neo4j_session: neo4j.Session, common_job_parameters: Di
     """
     run_cleanup_job('gcp_bigquery_cleanup.json', neo4j_session, common_job_parameters)
 
+@timeit
+def cleanup_gcp_bigquery_accesses(neo4j_session: neo4j.Session, common_job_parameters: Dict) -> None:
```

Comment:
Instead of creating two separate cleanup job add in existing

Reply:
done